kratos/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ set( KRATOS_CORE_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/find_nodal_h_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/skin_detection_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/processes/replace_elements_and_condition_process.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/processes/vtk_output_process.cpp
```

Comment:
I have a concern about if this is a process or an IO (gid_io is in includes folder instead of IO for legacy reasons, but maybe is a good opportunity to derive this class from IO and put it in the corresponding folder)

Reply:
The reasons why I did not do that is : 
- This is not exactly a IO ... It has only output functionality.

Reply:
Well, the gid_io is only O and the model_part_io was only I until 3 years ago

Reply:
I decided to derive it from IO and put it in the `input_output` folder. But for now one has to use the specified JSON input, since the common JSON input for output is not yet finalized.